clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -138,7 +133,13 @@ class AzureResourceManagerClient extends AzureBaseClient {
     ResourceManagementService.create(this.buildConfiguration(creds))
   }
 
-  static DeploymentExtended createTemplateDeploymentFromPath(
+  private static void createResourceGroupVNet(AzureCredentials creds, String resourceGroupName, String region) {
+    def vNetName = AzureUtilities.VNET_NAME_PREFIX + resourceGroupName
+
+    creds.getNetworkClient().createVirtualNetwork(creds, resourceGroupName, vNetName, resourceGroupName, "10.0.0.0/16")
```

Comment:
By the way, I've fixed the call createVirtualNetwork and will push it into azure-merge2. The PR should automatically pick it up